### DISCUSSION TITLE
Selling some boards

### DISCUSSION THREAD

**Discussion by GrauBlitz** [ole00/afterburner#32] [General]
Hello and thanks for this awesome project!

As always chinese fabricators want me to get atleast 5 to 10 boards produced. Is it okay to sell the unused PCBs online or is there some license in place to prevent this? I don't want to let them rot in my storage, so i just would bundle them with the other spare parts which i also got due to the package sizes (100 LEDs, 10 Switches, ...) and list them as a full set on ebay, if this is fine?

Thanks for the answers!

**Comment by ole00** [comment]
Thanks. Feel free to sell Afterburner boards or PCBs.

**Reply by GrauBlitz** [reply]
Thank you!
